app/javascript/shared/components/useKeyboardShortcuts.jsx

```diff
@@ -31,19 +31,24 @@ function isFormField(element) {
  *   "ctrl+alt+KeyG": (e) => {
  *     e.preventDefault();
  *     alert("Control Alt G has been pressed");
+ *   },
+  *   "KeyG~KeyH": (e) => {
+ *     e.preventDefault();
+ *     alert("G has been pressed quickly followed by H");
  *   },
  *   "?": (e) => {
  *     setIsHelpVisible(true);
  *   }
  * }
  *
- * useKeyboardShortcuts(shortcuts, someElementOrWindowObject);
+ * useKeyboardShortcuts(shortcuts, someElementOrWindowObject, {timeout: 1500});
  *
  * @param {object} shortcuts List of keyboard shortcuts/event
  * @param {EventTarget} [eventTarget=window] An event target.
+ * @param {object} [options = {}] An object for extra options
  *
  */
-export function useKeyboardShortcuts(shortcuts, eventTarget = window) {
+export function useKeyboardShortcuts(shortcuts, eventTarget = window, options = {}) {
```

Comment:
IIRC you can declare `options = { timeout: 500 }` in the params in JavaScript, right? This way the props declarations can also check they are the right params

Reply:
You're right, I've now changed that an made it cope with custom options a little better.